### DISCUSSION TITLE
Feature request: drafts and templates

### DISCUSSION THREAD

**Discussion by moshpirit** [Docile-Alligator/Infinity-For-Reddit#943] [Ideas]
One feature I really like from Slide for reddit is that if the app get closed or minimized, whatever we were writing gets saved as a draft.

I find it very practical and sometimes I even use it to create templates for conversations about certain topics that I ofter have with people. Although it's very poor designed because it doesn't allow us to manage the drafts and templates (eg: create folders to classify them).

I think it would be amazing to have two main folders: drafts (automatically saved responses that may be removed after a couple of days) and templates (that can stay forever and maybe be found as a text files in our device for easier edition/addition).

**Comment by Wladefant** [comment]
Easier to implement alternative is #944
